### PR TITLE
Add translation for api token

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: core/src/main/resources/jenkins/security/Messages_zh_CN.properties [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+ApiTokenProperty.DisplayName=API Token
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/security/Messages_zh_CN.properties]: Deliberate?

File: core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_zh_CN.properties [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  <li>\u79FB\u9664\u4ECE\u672A\u4F7F\u7528\u7684 Token</li>\
+  <li>\u5F53\u524D\u5728\u4F7F\u7528\u9057\u7559 Token \u7684\u7528\u6237\uFF0C\u4F7F\u7528\u65B0\u7CFB\u7EDF\u751F\u6210 Token\uFF0C\u5E76\u79FB\u9664\u4ED6\u4EEC\u7684\u9057\u7559 Token</li>\
+</ul>
+UserId=\u7528\u6237 ID
+UserFullName=\u7528\u6237\u5168\u79F0
+TokenName=Token \u540D\u79F0
+NumDaysSinceCreation=\u8DDD\u79BB\u521B\u5EFA\u65F6\u7684\u5929\u6570
+NumOfUse=# \u4F7F\u7528\u6570
+NumDaysSinceLastUse=\u8DDD\u79BB\u6700\u8FD1\u4E00\u6B21\u4F7F\u7528\u7684\u5929\u6570
+HasFreshToken=Fresh token?
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_zh_CN.properties]: Deliberate?

File: core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_zh_CN.properties [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+NoCreationDate=\u8BE5 Token \u6CA1\u6709\u521B\u5EFA\u65F6\u95F4
+NoCreationDateValue=\u672A\u77E5
+NoLastUse=\u8BE5 Token \u6CA1\u6709\u6700\u8FD1\u4F7F\u7528\u65F6\u95F4
+NoLastUseValue=\u4ECE\u672A\u4F7F\u7528
+RevokeAllSelected=\u79FB\u9664\u6240\u9009\u7684 Token
+RevokeAllSelected_confirm=\u4F60\u786E\u5B9A\u8981\u79FB\u9664\u6240\u9009 %num% \u7684 Token\u5417\uFF1F
+RevokeAllSelected_nothing=\u6CA1\u6709\u9009\u62E9 Token\uFF0C\u8BF7\u81F3\u5C11\u9009\u62E9\u4E00\u4E2A Token \u6765\u79FB\u9664
+Select=\u9009\u62E9
+only\ recent=\u4EC5 Recent
+only\ fresh=\u4EC5 Rresh
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_zh_CN.properties]: Typo

File: core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_zh_CN.properties [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+NoImpactedUser=\u6CA1\u6709\u7528\u6237\u4F7F\u7528\u9057\u7559 Token
+NoCreationDate=\u8BE5 Token \u6CA1\u6709\u521B\u5EFA\u65F6\u95F4
+NoCreationDateValue=\u672A\u77E5
+NoLastUse=\u8BE5 Token \u6CA1\u6709\u6700\u8FD1\u4F7F\u7528\u65F6\u95F4
+NoLastUseValue=\u4ECE\u672A\u4F7F\u7528
+RevokeAllSelected=\u79FB\u9664\u6240\u9009\u7684 Token
+RevokeAllSelected_confirm=\u4F60\u786E\u5B9A\u8981\u79FB\u9664\u6240\u9009 %num% \u7684 Token\u5417\uFF1F
+RevokeAllSelected_nothing=\u6CA1\u6709\u9009\u62E9 Token\uFF0C\u8BF7\u81F3\u5C11\u9009\u62E9\u4E00\u4E2A Token \u6765\u79FB\u9664
+Select=\u9009\u62E9
+only\ recent=\u4EC5 Recent
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_zh_CN.properties]: Deliberate?

**Comment by daniel-beck** [comment]
While "API token" may be a fixed technical term that is deliberately not translated, none of the other words are.

File: core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_zh_CN.properties [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+TokenName=Token \u540D\u79F0
+NumDaysSinceCreation=\u8DDD\u79BB\u521B\u5EFA\u65F6\u7684\u5929\u6570
+NumOfUse=# \u4F7F\u7528\u6570
+NumDaysSinceLastUse=\u8DDD\u79BB\u6700\u8FD1\u4E00\u6B21\u4F7F\u7528\u7684\u5929\u6570
+HasFreshToken=Fresh token?
+HasFreshToken_tooltip=\u4E00\u4E2A\u6700\u8FD1\u4F7F\u7528\u7684 Fresh Token \u662F\u7531\u65B0\u7CFB\u7EDF\u521B\u5EFA\u5E76\u6700\u8FD1\u4F7F\u7528\u4E86\u7684\uFF0C\u800C\u4E0D\u662F\u7528\u6237\u7684\u9057\u7559 Token. \n\
+  \u6CE8\u610F\uFF1A\u9057\u7559 Token \u7684\u521B\u5EFA\u65F6\u95F4\u5728\u6BCF\u6B21 Jenkins \u91CD\u542F\u540E\u4F1A\u88AB\u91CD\u7F6E\uFF0C\u4E5F\u5C31\u610F\u5473\u7740\u8BE5\u65F6\u95F4\u662F\u4E0D\u51C6\u786E\u7684\u3002
+HasFreshToken_ok_tooltip=\u8BE5\u7528\u6237\u6709\u4E00\u4E2AFresh Token
+HasFreshToken_warning_tooltip=\u8BE5\u7528\u6237\u6CA1\u6709 Fresh Token
+HasMoreRecentlyUsedToken=Recent token?
```
[the remark this answers is not in the document]
Comment (LinuxSuRen) [reply core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_zh_CN.properties]: Yeah, I thinks it should be keep original.

[the remark this answers is not in the document]
Comment (daniel-beck) [reply core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage_zh_CN.properties]: Neither 'recent' nor 'fresh' are fixed technical terms, so I'm surprised. keeping "API token" makes sense, perhaps, but those others? Not really.

**Comment by LinuxSuRen** [comment]
Got it.

**Review by daniel-beck — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**